cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -82,6 +82,8 @@ can be removed automatically by this program.`)
 	tokenFile = flag.String("credential_file", "", `If provided, this json file will be used to retrieve Service Account credentials.
 You may set the GOOGLE_APPLICATION_CREDENTIALS environment variable for the same effect.`)
 
+	ipAddressType = flag.String("ip_address_type", "PRIMARY", "Default to be PRIMARY. Options: PRIMARY / PUBLIC, PRIVATE")
```

Comment:
Can this instead be a list, so if we specified say PRIVATE,PRIMARY then if the instance has a private IP, it would use that, otherwise it would use the primary IP?